index.html

```diff
@@ -318,7 +323,13 @@ <h2>
       </h2>
       <p>
         This interface defines the state of an individual button on a gamepad
-        device.
+        device. A GamepadButton is a live object; that is, user interaction with
+        the button represented by the object will be reflected in the object's
+        properties. When the system receives new data from a connected gamepad,
+        a microtask is queued in the current event loop to update the state of
+        the GamepadButton object. If a gamepad device is disconnected, the
```

Comment:
The "If a gamepad device is disconnected..." should probably be it's own paragraph. Eventually, we need to turn that also into a queued task... "Upon disconnection of a gamepad, queue a task to... "

Reply:
Good idea. I uploaded a new commit where I've pulled out behavior that should be tied to system events (gamepad connection, gamepad disconnection, gamepad state update). Eventually these should be combined into an algorithm for each system event.